extension/index.js

```diff
@@ -49,4 +54,13 @@ $(() => {
     }
   }
 
-})
+  function viewDepsViz(e) {
+    const npmPkgName = e.data.pkg.name;
+    const windowFeatures = "resizable,scrollbars,status";
+
+    var otherWindow = window.open(`http://npm.anvaka.com/#/view/2d/${npmPkgName}`, "npmanvaka", windowFeatures);
```

Comment:
I'd be ok with this just being a simple `<a href>` instead, like all other links